From 9979f4e2b090b9a461d7171e1642486974d335dc Mon Sep 17 00:00:00 2001
From: Mamoru TASAKA <mtasaka@fedoraproject.org>
Date: Thu, 30 Nov 2023 14:28:27 +0900
Subject: [PATCH] FIX: support compilation with libxml2 2.12.0

libxml2 2.12.0 does some refactoring about functions prototypes places.
`xmlParseFile` definition is in `libxml/parser.h` anyway so include
this header file.

Closes #962 .
---
 src/modules/jackrack/jack_rack.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/modules/jackrack/jack_rack.c b/src/modules/jackrack/jack_rack.c
index f9b3ad40b..467ba99bc 100644
--- a/src/modules/jackrack/jack_rack.c
+++ b/src/modules/jackrack/jack_rack.c
@@ -29,6 +29,7 @@
 #include <ctype.h>
 
 #include <ladspa.h>
+#include <libxml/parser.h>
 #include <libxml/tree.h>
 
 #include "jack_rack.h"
